src/System.Management.Automation/engine/remoting/commands/PSRemotingCmdlet.cs

```diff
@@ -871,6 +874,10 @@ internal SSHConnection[] ParseSSHConnectionHashTable()
                     {
                         connectionInfo.KeyFilePath = paramValue;
                     }
+                    else if (paramName.Equals(PortParameter, StringComparison.OrdinalIgnoreCase))
```

Comment:
I think the Port parameter in the hash table should be an integer and not a string.  It would be more natural than enclosing the port value in quotes: @{ HostName = 'targetMachine'; UserName = 'UserA'; Port = 8080 }.  This would mean updating this parse method to special case the Port parameter.